src/main/java/edu/cornell/kfs/pdp/document/authorization/CuPayeeACHAccountMaintenanceDocumentAuthorizer.java

```diff
@@ -0,0 +1,27 @@
+package edu.cornell.kfs.pdp.document.authorization;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kuali.kfs.kim.api.identity.Person;
+import org.kuali.kfs.krad.document.Document;
+import org.kuali.kfs.pdp.document.authorization.PayeeACHAccountMaintenanceDocumentAuthorizer;
+
+public class CuPayeeACHAccountMaintenanceDocumentAuthorizer extends PayeeACHAccountMaintenanceDocumentAuthorizer {
+    private static final Logger LOG = LogManager.getLogger();
+
+    @Override
+    public boolean canApprove(Document document, Person user) {
+
+        try {
+            String documentInitializerPrincipalId = document.getDocumentHeader().getWorkflowDocument().getInitiatorPrincipalId();
+            if (user.getPrincipalId().equals(documentInitializerPrincipalId)) {
+                return false;
+            }
+        } catch (Exception ex) {
+            LOG.error("canApprove", ex.getMessage(), ex);
```

Comment:
You should adjust this logging statement to either directly concatenate the exception's message with the literal or to not manually print the exception message. Otherwise, Log4J might think you're preparing a parameterized message, as opposed to a custom message followed by the exception's stack trace.

Also, if an exception occurs during the is-initiator check, it might be safer to either return false or rethrow the exception, to ensure that the initiator can't take the action somehow.